src/python/WMCore/Services/WMStats/WMStatsReader.py

```diff
@@ -221,12 +220,10 @@ def _getLatestJobInfo(self, keys):
         """
         if len(keys) == 0:
             return []
-        options = {}
-        options["reduce"] = True
-        options["group"] = True
+        options = {"include_docs": True}
```

Comment:
I was going to ask how can we transfer less data if one of the couch views is asked to return the full document, but I believe the reason is that it was already returned by the call to `_getAllDocsByIDs` that you removed, right?

Reply:
yes _getAllDocsByIDs using include_docs so the data volume is less by {'_rev', 'timestamp'}